manifests/template.yaml

```diff
@@ -18,8 +19,10 @@ parameters:
 - name: SOURCE_NAME
   decsription: "The name of the VM to import (name or URL)"
   value: "http://192.168.42.1/my.ova"
-- name: SOURCE_LIBVIRT_URI
-  description: "(Optional) The URI to connect to the remote libvirt instance"
+- name: SOURCE_URI
+  description: "(Optional) The URI to connect to the remote instance"
+- name: PASS
```

Comment:
All in all I'd give the parameter names a round of refactoring to make them more self descriptive. nothing needed right now.

Reply:
Ack